app/javascript/src/components/editor/EditorItemCreate.svelte

```diff
@@ -24,30 +20,35 @@
     if (type == "item") $currentItem = newItem
   }
 
+  function withCloseDropdown(fn) {
+    return (... args) => {
+      fn(... args)
+      active = false
+    }
+  }
```

Comment:
This is kind of ugly, but I understand the purpose. A different option might be to simple add `active = false` to both `createTemplate` and `createItem`

Reply:
Either option looked ugly to me. But this one was more explicit, which is why I went with it.

Reply:
The reason I say this is ugly is because it takes more time to read and understand. While the other option is perhaps more repetition, at least you're able to understand it in an instant